setup.cfg

```diff
@@ -26,8 +26,8 @@ install_requires =
 
 [options.entry_points]
 console_scripts =
-    executable-name = asfsmd:main
+    asfmd = asfsmd:main
 
```

Comment:
Well spotted! Thanks @scottstanie 
By the way the executable name should be `asfsmd` (the same of the package name), do you agree?
Do you mind to amend the PR?

Reply:
Thanks